lib/influxdb2/client/flux_csv_parser.rb

```diff
@@ -32,27 +33,36 @@ def initialize(message, reference)
   end
 
   # This class represents Flux query error
+  #
   class FluxCsvParserError < StandardError
     def initialize(message)
       super(message)
     end
   end
 
   # This class us used to construct FluxResult from CSV.
+  #
   class FluxCsvParser
-    def initialize
+    include Enumerable
+    def initialize(response, stream: false)
```

Comment:
if is stream enabled then avoid storing all tables by `@tables[@table_index] = @table`